docs/changelog-entries/262.md

```diff
@@ -0,0 +1 @@
+- Merged `initializeData` into `initialize` [#262](https://github.com/precice/openfoam-adapter/pull/262).
```

Comment:
This should be in `changelog-entries/` (root directory), which is ignored by the linter.